lib/toucan.js

```diff
@@ -10,20 +10,127 @@ async function deployProxy(contractName, registry, args = []) {
 	return contract;
 }
 
-async function prepareToucanEnv() {
```

Comment:
any reason why you're not using typescript?

Reply:
just curious, why did you prefix the function name with an underscore?

Reply:
> any reason why you're not using typescript?

No reason, can be refactored to TS as part of a separate ticket/PR.

> just curious, why did you prefix the function name with an underscore?

Just copy-pasta from the monorepo.

Reply:
Which mono-repo? ^^